extensions/README.md

```diff
@@ -31,6 +31,7 @@ Current list of extended components:
 | [`amp-lightbox`](amp-lightbox/amp-lightbox.md)             | Allows for a “lightbox” or similar experience.                                              |
 | [`amp-list`](amp-list/amp-list.md)             | A dynamic list that can download data and create list items using a template |
 | [`amp-twitter`](amp-twitter/amp-twitter.md)               | Displays a Twitter Tweet.                                                                   |
+| [`amp-viafoura`](amp-viafoura/amp-viafoura.md)               | Generates a Viafoura Widget.                                                                   |
```

Comment:
The language is slightly inconsistent with other components: "Displays a Viafoura Engagement widgets" may be more appropriate??